src/pages/Main/Main.jsx

```diff
@@ -1,7 +1,244 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { FaChevronRight } from 'react-icons/fa';
+import { FaChevronLeft } from 'react-icons/fa';
+import { FaSortDown } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+import { FaRegHeart } from 'react-icons/fa';
 
-function Main(props) {
-  return <div>Main</div>;
+function Main() {
+  const [classList, setClassList] = useState([]);
+  useEffect(() => {
+    fetch('/data/mainClassList.json')
+      .then(res => res.json())
+      .then(data => setClassList(data));
+  }, []);
+
+  return (
+    <>
+      <SlideContainer>
+        <SlideBox>
+          <SlideImg src="https://cdn.class101.net/images/b9ff477c-a598-4222-b49e-23c5c85494af/2880xauto.webp" />
+          <ContentBox>
+            <TextBox>
+              <Title>
+                IOI 인기 클래스
+                <br />
+                구경하기
+              </Title>
+              <Detail>
+                IOI 아카데이에서
+                <br />
+                요즘 뜨는 트렌드 클래스를 둘러보세요!
+              </Detail>
+            </TextBox>
+            <SlideControlBox>
+              <SlideIndex>01</SlideIndex>
+              <SeparateIndex>|</SeparateIndex>
+              <SlideIndex>04</SlideIndex>
+              <StyleLine />
+              <FaChevronLeft className="IconLeft" size="20" color="white" />
+              <FaChevronRight className="IconRight" size="20" color="white" />
+            </SlideControlBox>
+          </ContentBox>
+        </SlideBox>
+      </SlideContainer>
+      <CategoryContainer>
+        <CategoryBox>
+          <CategoryName>메인 카테고리</CategoryName>
+          <FaSortDown />
+        </CategoryBox>
+        <CategoryBox>
+          <CategoryName>서브 카테고리</CategoryName>
+          <FaSortDown />
+        </CategoryBox>
+        <CategoryBox>
+          <CategoryName>인기순</CategoryName>
+          <FaSortDown />
+        </CategoryBox>
+      </CategoryContainer>
+      <ClassContainer>
+        {classList.map(product => {
+          return (
```

Comment:
```suggestion
        {classList.map(({id, coverImage, creator, classTitle, like, price}) => {
          return (
```
가독성과 추후 유지보수를 위해서 구조분해할당해서 적용해주세요!